tests/python/arith/test_arith_rewrite_simplify.py

```diff
@@ -88,6 +88,14 @@ class TestVector(BaseCompare):
         ),
         TestCase(y.astype("int32x2") + x.astype("int32x2"), (y + x).astype("int32x2")),
         TestCase(tvm.tir.Broadcast(0, 4) + y, tvm.tir.Broadcast(y, 4)),
+        TestCase(
+            tvm.tir.Broadcast(x, 4) + tvm.tir.Ramp(0, 1, tvm.tir.IntImm(dtype="int64", value=4)),
+            tvm.tir.Ramp(x, 1, 4),
+        ),
+        TestCase(
+            tvm.tir.Broadcast(x, tvm.tir.IntImm(dtype="int64", value=4)) + tvm.tir.Ramp(0, 1, 4),
```

Comment:
I actually meant that the value being broadcasted and ramp base being i64. but lanes remains i32, this would be a more common case in our setting

Reply:
ah okay, sorry for the confusion. I'll update the test cases to check that as well.

Reply:
@tqchen I've updated the test case, let me know if this looks good.